cfgov/unprocessed/apps/find-a-housing-counselor/css/hud_print.css

```diff
@@ -1,32 +1,48 @@
 /***** PRINT STYLES FOR FIND A HOUSING COUNSELOR *****/
+@media screen {
+    .u-print-only {
+        display: none;
+    }
+
+    .u-screen-only {
+        display: block;
+    }
+}
+
 
 @media print {
+    .u-screen-only {
+        display: none;
+    }
+    .u-print-only {
+        display: block;
+    }
+
     body {
         background: transparent !important;
         color: #000 !important; /* Black prints faster: h5bp.com/s */
         box-shadow: none !important;
         text-shadow: none !important;
     }
 
-    a,
-    a:link,
-    a:visited {
-        border: 0;
-        color: #000 !important; /* Black prints faster: h5bp.com/s */
-        text-decoration: underline;
-    }
-
+    /* Spell out abbreviations. */
     abbr[title]:after {
-        content: " (" attr(title) ")";
+        content: ' (' attr(title) ')';
     }
 
     /*
-     * Don't show links for images, or javascript/internal links
+     * Don't show links.
      */
+    a:after {
+        content: '';
+    }
 
-    a[href^="javascript:"]:after,
-    a[href^="#"]:after {
-        content: "";
+    a,
+    a:link,
```

Comment:
I didn't think so. A printed page won't be clickable 😉

Reply:
Mostly I was noticing that the old print styles left links underlined, and it looks like our general print styles leave links underlined elsewhere on the site, so I was thinking we might want to keep the underlines. But, given that someone has likely chosen to specifically *print* this page rather than make a PDF (otherwise they probably would have used the PDF button), I'm fine keeping the underlines hidden.